xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1497,7 +1497,7 @@ void CGUIWindowVideoBase::LoadPlayList(const std::string& strPlayList, int iPlay
     // load it
     if (!pPlayList->Load(strPlayList))
     {
-      CGUIDialogOK::ShowAndGetInput(6, 0, 477, 0);
+      CGUIDialogOK::ShowAndGetInput(6, 477);
```

Comment:
> @da-anda
> Can you check out string ID 6?
> This one is really weird and i've never really seen it actually in XBMC/Kodi itself
> https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L53
> 
> Edit:
> It used to be "XBMC SVN" and it should show as dialog header

See http://trac.kodi.tv/ticket/14128 that pops up in http://trac.kodi.tv/attachment/ticket/14128/screenshot009.png

I was going to remove that string in some later PR but not sure if I did what would come up instead when that error happens.